Homework_11/1.py

```diff
@@ -0,0 +1,83 @@
+
+"""
+Создайте класс Матрица. Добавьте методы для:
+○ вывода на печать,
+○ сравнения,
+○ сложения,
+○ *умножения матриц
+"""
+
+
+class Matrix:
+
+    def __init__(self, matr):
+        self._matr = matr
+
+    def get_matrix(self):
+        return self._matr
+
+    def __add__(self, other):
+        if len(self._matr) != len(other._matr) or len(self._matr[0]) != len(other._matr[0]):
+            return f'Error: матрицы разных размеров'
+        else:
+            return Matrix([[self._matr[i][j] + other._matr[i][j] for j in range(len(self._matr[0]))] for i in range(len(self._matr))])
+
+    def __mul__(self, other):
+        if len(self._matr[0]) != len(other._matr):
+            return f'Error: невозможно перемножить матрицы'
+        else:
+            new_matr = [[sum(i * j for i, j in zip(i_row, j_col)) for j_col in zip(*other._matr)] for i_row in self._matr]
```

Comment:
else, кстати не нужен
```suggestion

        new_matr = [[sum(i * j for i, j in zip(i_row, j_col)) for j_col in zip(*other._matr)] for i_row in self._matr]
```